app/src/main/java/com/codesoom/assignment/domain/UserRepository.java

```diff
@@ -2,14 +2,44 @@
 
 import java.util.Optional;
 
+/**
+ * 회원 데이터를 다루는 명령을 정의합니다.
+ */
 public interface UserRepository {
     User save(User user);
 
+    /**
+     * 해당 이메일의 회원 유무를 리턴합니다.
+     * @param email 회원 이메일
+     * @return 회원 유무
+     */
     boolean existsByEmail(String email);
 
+    /**
+     * 해당 식별자의 회원 유무를 리턴합니다.
+     * @param id 회원 식별자
+     * @return 회원 유무
+     */
+    boolean existsById(Long id);
```

Comment:
`boolean`을 리턴해주는 메소드는 javadoc을 강하고 분명하게 적어주는 것이 좋습니다.
```suggestion
    /**
     * 등록된 회원이라면 true, 등록된 회원이 아니라면 false를 리턴합니다.
     *
     * @param id 회원 식별자
     * @return 등록된 회원인 경우에만 true
     */
    boolean existsById(Long id);
```